src/types/refs.rs

```diff
@@ -280,6 +281,16 @@ impl RefCount {
     fn weak_refs(&self) -> &AtomicU32 {
         unsafe { AtomicU32::from_ptr(&self.0.weakRefs as *const _ as _) }
     }
+
+    fn new() -> PoolRef<Self> {
+        let mut refcount = RefCount::alloc().expect("should allocate a RefCount");
+        let ptr = refcount.as_mut_ptr();
+        unsafe {
+            (*ptr).0.strongRefs = 1;
+            (*ptr).0.weakRefs = 1;
```

Comment:
```suggestion
            (*ptr).0.weakRefs = 0;
```
should probably be zero